composer.json

```diff
@@ -26,6 +26,7 @@
         "ext-json": "*"
     },
     "require-dev": {
+        "larapack/dd": "^1.1",
```

Comment:
Why add the `larapack/dd`? There is no `dd()` call?

Reply:
It's a dev dependency for debugging

Reply:
It's a standard across Spatie packages. This way we can use `dd` when working on the package.